omopetl/transform.py

```diff
@@ -83,6 +89,26 @@ def transform_normalize_date(self, source_column, target_column, transformation)
         date_format = transformation.get("format", "%Y-%m-%d")
         return pd.to_datetime(self.data[source_column], errors="coerce").dt.strftime(date_format)
 
+    def transform_filter(self, source_column, target_column, transformation):
```

Comment:
This function was removed in https://github.com/tompollard/omopetl/pull/21. It looks like an outdated main branch has been merged into this branch.